cmake/CcacheVersion.cmake

```diff
@@ -11,7 +11,9 @@
 #    version_info has not been substituted). In this case we fail the
 #    configuration.
 # 3. Building from a Git repository. In this case the version will be a proper
-#    version if building a tagged commit, otherwise "branch.hash(+dirty)".
+#    version if building a tagged commit, otherwise "branch.hash(+dirty)". In
+#    case where git is not available, version will be 'unknown' (e.g. mounting
+#    your source directory into docker without installing git).
```

Comment:
```suggestion
#    version if building a tagged commit, otherwise "branch.hash(+dirty)". In
#    case Git is not available, the version will be "unknown".
```